frontend/src/component/common/ProFeatureTooltip/ProFeatureTooltip.tsx

```diff
@@ -29,17 +26,43 @@ const StyledLink = styled(Link)(({ theme }) => ({
     width: 'fit-content',
 }));
 
-export const ProFeatureTooltip = ({ children }: ProFeatureTooltipProps) => {
+export interface ProFeatureTooltipProps {
+    children: React.ReactNode;
+    title: string;
```

Comment:
Instead of repeating ourselves maybe we could have an approach like:
```JS
export enum FeatureLevel {
    PRO = 'Pro & Enterprise feature',
    ENTERPRISE = 'Enterprise feature',
}

//...

export interface ProFeatureTooltipProps {
    children: React.ReactNode;
    level: FeatureLevel;
    origin: string;
    center?: boolean;
}

//...

<StyledTitle>
    <ProPlanIcon />
    {level}
</StyledTitle>
```

Or we can also type `level` as `'pro' | 'enterprise'` and have a ternary for the title as a simpler alternative.